### DISCUSSION TITLE
change theme on the fly

### DISCUSSION THREAD

**Discussion by jcastroa87** [Studio-42/elFinder#3582] [Q&A]
hello everyone, i want to change theme between light and dark on execution time, today i have this configuration:

```
var loadTheme = 'default';

var elf = $('#elfinder').elfinder({
themes: {
    default : 'https://cdn.jsdelivr.net/gh/RobiNN1/elFinder-Material-Theme/manifests/material-gray.json',
    dark : 'https://cdn.jsdelivr.net/gh/RobiNN1/elFinder-Material-Theme/manifests/material-default.json',
},
theme: loadTheme
}).elfinder('instance');
```

i want to change between "default" and "dark" without reloading page, is thats possible? someone can give me any advice how to do it?

Thanks

**Comment by nao-pon** [comment]
@jorgetwgroup You can do that in the Theme under Preferences on the toolbar gear icon.

And the code for that function is the following part:
https://github.com/Studio-42/elFinder/blob/98d48fa9877a1905a0c36b8fda908b7193614b9c/js/commands/preference.js#L109-L110

**Comment by jcastroa87** [comment]
Thanks i will try.
